src/slack.js

```diff
@@ -708,7 +708,7 @@ async function storeOrder(order, items) {
       'Count': item.count,
       ...(item.price ? {Price: item.price} : null),
       'Order': [airtableOrder.getId()],
-      'Verified Price': !item.withoutLogin && item.price,
+      'Verified Price': !item.withoutLogin && (item.price !== null),
```

Comment:
In other places we check if item.price is not NaN, so this will probably not be enough. Please use Boolean(item.price) instead, I doubt there will be something for free on alza :smile: 
```suggestion
      'Verified Price': !item.withoutLogin && Boolean(item.price),
```